### PR TITLE
Add a bunch of framework packages dependent on pyobjc-core and the gr…

### PR DESCRIPTION
**Pull request by kastman**
…aphics frameworks

### CODE REVIEW

**Comment by kastman** [comment]
Or, surprisingly, it just won't even attempt to build until all the dependencies resolve and this turns green!

File: recipes.bkup/pyobjc-framework-accounts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{% set version = "3.2.1" %}
+{% set md5 = "dbdf7f70fdc9e70a6e9b27f198664800" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/87/62/ac611070f3f880556fe543916e38813389222e7f37f2f391a062b8486ec8/{{ name }}-{{version}}.tar.gz
```
Comment (ocefpaf) [inline recipes.bkup/pyobjc-framework-accounts/meta.yaml]: Please change that URL to a more readable form. See the example in this repo.

File: recipes.bkup/pyobjc-framework-avkit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+about:
+  home: http://pyobjc.sourceforge.net
+  license: MIT
+  summary: 'Wrappers for the framework AVKit on Mac OS X'
+  license_family: MIT
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (ocefpaf) [inline recipes.bkup/pyobjc-framework-avkit/meta.yaml]: Please remove the unnecessary comments.

File: recipes.bkup/pyobjc-framework-applescriptkit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    - pyobjc-core >=3.2.1
+    - pyobjc-framework-cocoa >=3.2.1
+
+  run:
+    - python
+    - pyobjc-core >=3.2.1
+    - pyobjc-framework-cocoa >=3.2.1
+
+test:
+  # Python imports
```
Comment (ocefpaf) [inline recipes.bkup/pyobjc-framework-applescriptkit/meta.yaml]: Please remove the comment.

**Pull request closed without merging**